CHANGES.md

```diff
@@ -1,5 +1,35 @@
 # Changes
 
+## stable-2.12.4
+
+This stable release fixes a memory leak in the Destination controller, and also
+includes other bug fixes for the Linkerd control plane, CLI, and extensions.
+
+* CLI
+  * Fixed an issue in the CLI where `--identity-external-ca` would set an
+    incorrect field (thanks @anoxape!)
+
+* Control Plane
+  * Fixed an issue in the destination controller's cache that could result in
+    stale endpoints when using EndpointSlice objects
+  * Fixed an issue where control plane components could fail to start on large
+    clusters because of failing readiness probes while caches were being
```

Comment:
@hawkw Re-reading the PR that this fixes, I believe this cause is failing `liveness` probes, rather than `readiness` probes.

https://github.com/linkerd/linkerd2/pull/10166

cc @adleong

Reply:
I didn't intend to select four lines! I just meant 15 and 16:

> Fixed an issue where control plane components could fail to start on large
    clusters because of failing readiness probes while caches were being

Reply:
@jeremychase ah, I think you're right --- the title of the PR started with "readiness probes" so i misinterpreted it a bit. good catch!

Reply:
okay, i updated this one and also tried to reword the changelog entry to be a bit more readable.